src/components/accordion/Accordions.js

```diff
@@ -1,31 +1,40 @@
-import Accordion from '@mui/material/Accordion'
-import AccordionSummary from '@mui/material/AccordionSummary'
-import AccordionDetails from '@mui/material/AccordionDetails'
 import { Typography } from '@mui/material'
 import { useTranslation } from 'react-i18next'
 import { Box } from '@mui/system'
-import { style } from './accordion.styles'
+import Accordion from '@mui/material/Accordion'
+import AccordionSummary from '@mui/material/AccordionSummary'
+import AccordionDetails from '@mui/material/AccordionDetails'
+import ExpandMoreRoundedIcon from '@mui/icons-material/ExpandMoreRounded'
 
-const Accordions = ({ items, onChange, activeIndex, styles }) => {
+import { style } from '~/components/accordion/accordion.styles'
+
+const Accordions = ({ items, onChange, activeIndex, showMoreIcon, square }) => {
   const { t } = useTranslation()
 
+  const accordionType = showMoreIcon ? 'withShowMoreIcon' : 'noShowMoreIcon'

```

Comment:
Maybe instead of in every place putting "style[accordionType]" you can create here style variable and just use it in all places?

const accordionStyle = style[showMoreIcon ? 'withShowMoreIcon' : 'noShowMoreIcon']

...

Reply:
done